riak_test/tests/riak_cs_debug_test.erl

```diff
@@ -0,0 +1,170 @@
+%% ---------------------------------------------------------------------
+%%
+%% Copyright (c) 2007-2015 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% ---------------------------------------------------------------------
+
+-module(riak_cs_debug_test).
+
+-compile(export_all).
+-export([confirm/0]).
+-include_lib("eunit/include/eunit.hrl").
+
+-define(assertContainsAll(ExpectedList,ActualList),
+        lists:foreach(
+          fun(X) -> ?assert(lists:member(X, ActualList)) end,
+          ExpectedList)).
+
+-define(assertMatchAny(Pattern, ActualList),
+        ?assert(
+           lists:any(
+             fun(X) ->
+                     case re:run(X, Pattern) of
+                         {match, _} -> true;
+                         nomatch -> false
+                     end
+             end, ActualList))).
+
+-define(assertNotMatchAny(Pattern, ActualList),
+        ?assert(
+           lists:all(
+             fun(X) ->
+                     case re:run(X, Pattern) of
+                         {match, _} -> false;
+                         nomatch -> true
+                     end
```

Comment:
Ah, revert my comment above. The current macro exhibit better correspondence with `assertMatchAny`. Sorry for noice.